docs/product-user-guide/src/main/asciidoc/project-packaging-con.adoc

```diff
@@ -0,0 +1,6 @@
+[id='project-packaging-con_{context}']
+= Packaging a project
+
+In {PRODUCT}, each project is packaged as a Knowledge JAR (KJAR) file and deployed to a {KIE_SERVER} that runs the service from that KJAR file. Project KJAR files are stored in a Maven repository and identified by three values: `GroupId`, `ArtifactId`, and `Version` (GAV). The `Version` value must be unique for every new version that might need to be deployed. To identify an artifact (including a KJAR file), you need all three GAV values.
```

Comment:
1st instance of "service" used a shortened version of "decision service" or should be replaced by "KIE container"